_config.yml

```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-slate
```

Comment:
This file should probably be in gitignore and not part of the repo

Reply:
I clicked something in the github interface and I guess it added this. I removed it entirely